src/api/v1/employees/routes/delete.ts

```diff
@@ -0,0 +1,14 @@
+import { FastifyReply, FastifyRequest } from 'fastify'
+
+export default async (req: FastifyRequest<{Body: { id: string } }>, res: FastifyReply): Promise<void> => {
+  if (!req.body.id) {
+    await res.status(400).send({ message: 'Field cannot be empty' })
```

Comment:
Slightly unintuitive error message.
```suggestion
    await res.status(400).send({ message: 'ID field cannot be empty' })
```